src/mgmt/operator/obj/droplet.py

```diff
@@ -36,9 +36,11 @@ def __init__(self, name, obj_api, opr_store, spec=None):
         super().__init__(name, obj_api, opr_store, spec)
         self.kind = "Droplet"
         self.plural = "droplets"
-        self.ip = ""
-        self.mac = ""
-        self.phy_itf = 'eth0'
+        self.ip = []
+        self.mac = []
+        self.zgc_id = ''
+        self.phy_itf = ''
+        self.name = name
```

Comment:
We can remove this as the call to super already takes care of this

Reply:
fixed